sci-electronics/kicad/kicad-5.0.1.ebuild

```diff
@@ -0,0 +1,133 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit check-reqs cmake-utils eapi7-ver eutils gnome2-utils python-single-r1 toolchain-funcs wxwidgets xdg
+
+DESCRIPTION="Electronic Schematic and PCB design tools"
+HOMEPAGE="http://www.kicad-pcb.org"
+SRC_URI="https://launchpad.net/${PN}/$(ver_cut 1-2)/${PV}/+download/${P}.tar.xz"
+
+LICENSE="GPL-2+ GPL-3+ Boost-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="doc examples github +ngspice occ +oce openmp +python"
+
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+	?? ( occ oce )
+"
+
+COMMON_DEPEND=">=x11-libs/wxGTK-3.0.2:${WX_GTK_VER}[X,opengl]
+	python? (
+		dev-python/wxpython:${WX_GTK_VER}[opengl,${PYTHON_USEDEP}]
+		${PYTHON_DEPS}
+	)
+	>=dev-libs/boost-1.61[context,nls,threads,python?,${PYTHON_USEDEP}]
+	media-libs/glew:0=
+	media-libs/glm
+	media-libs/freeglut
+	media-libs/mesa
+	ngspice? (
+		sci-electronics/ngspice[shared]
+	)
+	occ? ( >=sci-libs/opencascade-6.8.0:* )
+	oce? ( sci-libs/oce )
+	>=x11-libs/cairo-1.8.8
+	>=x11-libs/pixman-0.30"
+DEPEND="${COMMON_DEPEND}
+	doc? ( app-doc/doxygen )
+	github? ( net-misc/curl[ssl] )
+	python? ( >=dev-lang/swig-3.0:0 )"
+RDEPEND="${COMMON_DEPEND}
+	sci-electronics/electronics-menu
+"
+CHECKREQS_DISK_BUILD="800M"
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+	use openmp && tc-check-openmp
+	setup-wxwidgets
+	check-reqs_pkg_setup
+}
+
+src_prepare() {
+	xdg_src_prepare
+	cmake-utils_src_prepare
+
+	epatch "${FILESDIR}"/"${PN}-5.0.0-curl.patch"
```

Comment:
You can use `PATCHES` array, and call `default`, or even `eapply` to prefer over this old epatch.

Reply:
Again, this code was added by Gentoo devs merging my code. I will fix this as well.